protoutils/protoutils_test.go

```diff
@@ -332,24 +332,33 @@ func TestStructToStructPb(t *testing.T) {
 	}
 }
 
+func TestStructToStructPbOmitEmpty(t *testing.T) {
+	expected := map[string]interface{}{"x": 0.0, "y": 0.0}
+	t.Run("StructToStructPBIgnoreOmitEmpty includes all values", func(t *testing.T) {
+		data, err := StructToStructPbIgnoreOmitEmpty(OmitStruct{})
```

Comment:
I think I'm missing something; where is OmitStruct defined?

Reply:
https://github.com/viamrobotics/goutils/blob/c6d1b7e6309fff9e05367c8c0f0628a6dcb74c88/protoutils/protoutils_test.go#L372 Its at the bottom of the file. It just has a field that is tagged as omitEmpty.